_integration-schemas/zoom/v1/meeting_poll_results.md

```diff
@@ -0,0 +1,44 @@
+---
+tap: "zoom"
+version: "1"
+key: ""
+
+name: "meeting_poll_results"
+doc-link: ""
+singer-schema: "https://github.com/singer-io/tap-zoom/blob/master/tap_zoom/schemas/meeting_poll_results.json"
+description: |
+  The `{{ table.name }}` table contains information about the results of polls from your {{ integration.display_name }} meetings.
```

Comment:
Zoom's API docs have some specifications for how the endpoint powering this table works:

```
- Host user type must be Pro.
- Meeting must be a scheduled meeting. Instant meetings do not have polling features enabled.
```

I'd work this into the table description, or folks might wonder why data isn't available for certain meetings.